src/index.ts

```diff
@@ -0,0 +1,4 @@
+import { serve } from '@hono/node-server'
+import { userRoute } from './routes'
+
```

Comment:
Please read about Group routing and base path
https://hono.dev/api/routing#base-path https://hono.dev/api/routing#grouping